src/Umbraco.Web.Common/RuntimeMinification/SmidgeOptionsSetup.cs

```diff
@@ -12,11 +13,25 @@ public SmidgeOptionsSetup(IOptions<RuntimeMinificationSettings> runtimeMinificat
         => _runtimeMinificatinoSettings = runtimeMinificatinoSettings;
 
     /// <summary>
-    ///     Configures Smidge to use in-memory caching if configured that way or if certain cache busters are used
+    ///     Configures Smidge to use in-memory caching if configured that way or if certain cache busters are used. As well as the cache buster type.
     /// </summary>
     /// <param name="options"></param>
     public void Configure(SmidgeOptions options)
-        => options.CacheOptions.UseInMemoryCache = _runtimeMinificatinoSettings.Value.UseInMemoryCache ||
+    {
+        options.CacheOptions.UseInMemoryCache = _runtimeMinificatinoSettings.Value.UseInMemoryCache ||
```

Comment:
Maybe whilst we are in the vicinity we could fix the typo in this variable name?  `_runtimeMinificatinoSettings` should be `_runtimeMinificationSettings`.  Unfortunately you'll have to leave `runtimeMinificatinoSettings` provided in the constructor as changing that is theoretically breaking (i.e. it's unlikely, but just in case anyone is creating this class in code and using named parameters).